repose-aggregator/services/phone-home/src/main/scala/org/openrepose/core/services/phonehome/PhoneHomeService.scala

```diff
@@ -127,7 +128,23 @@ class PhoneHomeService @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_VERSI
         }
       }
 
+      def getCurrentFormattedDateTime(): String = {
+        // From http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
+        // "For formatting, if the offset value from GMT is 0, 'Z' is produced."
+        // This only manipulates the Time Zones that produce a 'Z'.
+        val formattedString = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX").format(new Date())
+        if (formattedString.endsWith("Z")) {
+          formattedString.substring(0, formattedString.length() - 1) + "+00:00"
+        } else {
+          formattedString
+        }
+      }
+
       val updateMessage = Json.stringify(Json.obj(
+        "createdAt" -> getCurrentFormattedDateTime,
+        "createdAtMillis" -> System.currentTimeMillis,
+        "jreVersion" -> System.getProperty("java.version", "UNKNOWN"),
+        "jvmName" -> System.getProperty("java.vm.name", "UNKNOWN"),
         "serviceId" -> originServiceId,
```

Comment:
True, but depending on your query, you may still have a bunch of results from different services. For example, say I found a bug in the api-validator filter. Now I want to see every service which uses the filter, so I query mongo. I get back a bunch of results, but I have to look at the 5th line of each to find what service it belongs to rather than the 1st. Or, more likely, I'm looking at compact JSON so I have to actually parse things with my eyeballs.

I'm actually not certain of the format the Mongo presents data in, or how good the tools are that will query the database, but this seemed like a simple thing to do that may or may not help.